src/test/java/com/navfort/pages/VehiclesPage.java

```diff
@@ -0,0 +1,18 @@
+package com.navfort.pages;
+
+import com.navfort.utilities.Driver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class VehiclesPage extends QuickLaunchpadPage {
```

Comment:
Add your initials to the name of the class VehiclesPage_AS (on of David's prerequisites)